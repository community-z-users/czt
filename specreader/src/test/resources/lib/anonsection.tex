\begin{zed}
a : \arithmos
\end{zed}
\begin{zsection}
\SECTION anonsection
\end{zsection}
\begin{zed}
a : \arithmos
\end{zed}
