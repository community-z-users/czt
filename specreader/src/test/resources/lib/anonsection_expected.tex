net.sourceforge.czt.specreader.IllegalAnonSectionException: Formal text should not precede named section in /lib/anonsection.tex